test/end-to-end/stock/stock.movements.spec.js

```diff
@@ -30,23 +30,23 @@ function StockMovementsRegistryTests() {
     await modal.switchToDefaultFilterTab();
     await modal.setPeriod('allTime');
     await modal.submit();
-    await GU.expectRowCount(gridId, 18 + (2 * depotGroupingRow));
+    await GU.expectRowCount(gridId, 20 + (2 * depotGroupingRow));
```

Comment:
Collisions are coming!  🗡️ 

The MySQL8 fixes are going to clobber all this, but hopefully we can sort it out and have a better body of tests afterwards.